apps/api/run-seeder-tests-sequential.sh

```
@@ -0,0 +1,130 @@
+NC='\033[0m' # No Color
+
+# Track results
+PASSED=0
+FAILED=0
+SKIPPED=0
+FAILING_TESTS=()
+
+# Function to run a test and track results
+run_test() {
+    local test_file="$1"
+    local test_name="$(basename $test_file)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)